reference/git_github.rst

```diff
@@ -3,17 +3,6 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
-Best practises
```

Comment:
It could, but I expect that `bestpractises.rst` will cover more than just git & Github topic. I'll add a link.